doc/Extensions/MIB-based-polling.md

```diff
@@ -69,9 +69,9 @@ The components involved in MIB-based polling are:
 
   - During discovery, relevant MIBs are parsed using `snmptranslate`, and the
     data returned is used to populate a database which guides the poller in
-    what to store.  At the moment, only OIDs of INTEGER, Integer32, Gauge32,
```

Comment:
I've updated the documentation in another PR, so best to just remove d3cddffdc7d47a717813fd8de66b3667581c3be4 from this PR.